modules/covid_vaccine/spec/models/covid_vaccine/v0/raw_form_data_spec.rb

```diff
@@ -63,5 +63,24 @@
           .to eq(["Vaccine interest can't be blank"])
       end
     end
+
+    context 'without presence of birth_date' do
+      let(:attributes) { { email: 'jane.doe@email.com', zip_code: '12345-1234',
+                           vaccine_interest: 'yes', birth_date: '' } }
+
+      it 'is not valid without the presence of vaccine_interest' do
```

Comment:
the label on this spec is wrong (copy and paste), but LGTM

Reply:
Fixed